Tools/create-release.py

```diff
@@ -1,23 +1,18 @@
 #!/usr/bin/env python2.7
 
 import semver
-import xml.etree.ElementTree as ET
 import sys 
 
-ANDROID_STRINGS_XML = "AndroidSDK/res/values/strings.xml"
+SDK_VERSION_FILE = "AndroidSDKCore/sdk-version.txt"
 
-def get_current_version(root):
-  for element in root.iter("string"):
-    if element.attrib.get("name") == "sdk_version":
-      return element.text
+def get_current_version():
```

Comment:
This super simple, we probably could have explored this kinda solution from the onset.

Reply:
It's only easy because of `file('sdk-version.txt').text` which neither of us knew about at the time ;-)